src/llmq/quorums_dkgsession.cpp

```diff
@@ -1278,6 +1278,12 @@ std::vector<CFinalCommitment> CDKGSession::FinalizeCommitments()
             logger.Batch("failed to recover quorum sig");
             continue;
         }
+
+        if (!fqc.Verify(pindexQuorum, true)) {
+            logger.Batch("failed to verify final commitment");
+            continue;
+        }
+
```

Comment:
Let's move this below `t2.stop();`, wrap in its own timer and tweak the log below accordingly

Reply:
Makes sense 👍 See 7621990